### DISCUSSION TITLE
Unable to import boilerplate

### DISCUSSION THREAD

**Discussion by TheRealKwabena** [DevSkillsHQ/discusssions#20] [Help] [answered]
I tried to clone the repository but I was not able to do so. I am using Linux by the way

**Comment by fenske** [comment, marked as answer]
Hi @TheRealKwabena. I'm Anton from Alva. Happy to help.

While I need to see the error to pinpoint the problem, there is a high chance you need to configure your SSH key. Check out [this article](https://www.notion.so/I-can-t-clone-my-assignment-repository-749b58bbd7d9449a91aa3a091bcf6590) that walks you through the process. 

And then there is also the HTTP approach described in the article above.

Hope this helps.

Let me know if you have any other questions.
